plugin/src/main/scala/com/geirsson/CiReleasePlugin.scala

```diff
@@ -25,29 +25,34 @@ object CiReleasePlugin extends AutoPlugin {
   override def requires =
     JvmPlugin && SbtPgp && DynVerPlugin && GitPlugin && Sonatype
 
-  def isTravisSecure: Boolean =
+  def isSecure: Boolean =
```

Comment:
Can we keep the old Travis methods, forward them to the new methods and add `@deprecated`?

Reply:
@olafurpg good idea. Done! For the since field of `deprecated` I went we the next incremental release number from the last release. If you prefer a different next release number or a date instead let me know. Thanks!